operator/builtin/input/file/config.go

```diff
@@ -117,7 +119,8 @@ func (c InputConfig) Build(context operator.BuildContext) ([]operator.Operator,
 		return nil, err
 	}
 
-	splitFunc, err := c.Multiline.Build(context, encoding.Encoding, false)
+	// Ensure that multiline is buildable
+	_, err = c.Multiline.Build(encoding.Encoding, false, nil)
```

Comment:
I'm trying to figure out why `flushAtEOF` wouldn't solve this problem. Likely I am missing something, but it seems like this should take care of it.

Of course, we don't want this enabled by default, but if it would work, then we should be able to solve this by just adding a boolean parameter and passing it through here.

Thoughts on this?

Reply:
In case log is written in multiple parts, `flushAtEOF` will result in flushing every part separately. [link to test](https://github.com/open-telemetry/opentelemetry-log-collection/blob/main/operator/builtin/input/file/file_test.go#L327-L344) I think we are replacing one edge case with another one

If I set `flushAtEOF` to `true` for file input, file is re-read every time due to some fingerprinting issue, which probably needs to be investigated.

Reply:
I tested it with `fingerprint_size: 20` and what i observed for this config
```
receivers:
  filelog:
    start_at: beginning
    include:
      - ./open-receiver/test.log
    multiline:
      line_start_pattern: 'START '
      # force_flush_period: 3s
    fingerprint_size: 20

exporters:
  logging:
    loglevel: debug

service:
  pipelines:
    logs:
      receivers:
        - filelog
      exporters:
        - logging
```

adding log to the file (copying and pasting `asd\n`) results sometimes with `asd` log, sometimes with  `asd\nasd`

Reply:
Thanks for clarifying.